### DISCUSSION TITLE
Issue on compiling

### DISCUSSION THREAD

**Discussion by ghost** [YoshiCrafter29/YoshiCrafterEngine#296] [Q&A]
Anyone know how to fix this?

source/flixel/FlxSprite.hx:1081: characters 18-34 : Null<flixel.math.FlxRect> has no field getRotatedBounds
source/flixel/FlxSprite.hx:1053: characters 18-34 : Null<flixel.math.FlxRect> has no field getRotatedBounds
source/flixel/FlxSprite.hx:1047: characters 20-36 : Field getRotatedBounds is declared 'override' but doesn't override any field

**Comment by yahyamodder** [comment]
you don't need to compile
